bundles/org.openhab.binding.shelly/README.md

```diff
@@ -1178,6 +1178,33 @@ Refer to [Smartify Roller Shutters with openHAB and Shelly](doc/UseCaseSmartRoll
 |       | totalKWH     | Number   | yes       | Total energy consumption in kwh since the device powered up (resets on restart)   |
 |       | lastUpdate   | DateTime | yes       | Timestamp of the last measurement                                                 |
 
+### Shelly Plus Dimmer 10v(thing-type: shellyplus10v)
```

Comment:
Please a space just before the parenthesis

Reply:
fixed